Buddies/Onboarding/SignUpInfoVC.swift

```diff
@@ -63,6 +65,7 @@ class SignUpInfoVC: LoginBase, UIImagePickerControllerDelegate, UINavigationCont
         // Do any additional setup after loading the view.
         bioText.delegate = self
         bioText.textColor = UIColor.lightGray
+        bioText.layer.cornerRadius = 5
```

Comment:
I think the bio-text field looks nicer with a little padding.  

`bioText.textContainerInset = UIEdgeInsets(top: 12, left: 7, bottom: 12, right: 7)`